internal/network/command_delete.go

```diff
@@ -0,0 +1,57 @@
+package network
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/errors"
+	"github.com/confluentinc/cli/v3/pkg/examples"
+	"github.com/confluentinc/cli/v3/pkg/form"
+	"github.com/confluentinc/cli/v3/pkg/output"
+	"github.com/confluentinc/cli/v3/pkg/resource"
+)
+
+func (c *command) newDeleteCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "delete <id>",
+		Short: "Delete a network.",
+		Args:  cobra.ExactArgs(1),
+		// TODO: Implement autocompletion after List Network is implemented.
+		// ValidArgsFunction: pcmd.NewValidArgsFunction(c.validArgs),
+		RunE: c.delete,
+		Example: examples.BuildExampleString(
```

Comment:
Feel free to delete this example, since it doesn't use any flags and is just as helpful as the usage string, IMO.

Reply:
Makes sense! The example seems redundant. Thank you!